<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewInstanceDialog</class>
 <widget class="QDialog" name="NewInstanceDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Instance</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/toolbar/new</normaloff>:/icons/toolbar/new</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="iconBtnLayout">
     <item>
      <spacer name="iconBtnLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="iconButton">
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="iconBtnRightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="instNameTextBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelVersion_3">
       <property name="text">
        <string>&amp;Group:</string>
       </property>
       <property name="buddy">
        <cstring>groupBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="versionTextBox">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="btnChangeVersion">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QRadioButton" name="modpackBox">
       <property name="text">
        <string>Import &amp;Modpack (local file or link):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="modpackBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLineEdit" name="modpackEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>http://</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QRadioButton" name="versionBox">
       <property name="text">
        <string>Vani&amp;lla Minecraft (select version):</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
       <item>
        <widget class="QRadioButton" name="importBtn">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="importProviderBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QComboBox" name="importCandidateBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>instNameTextBox</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>versionBox</tabstop>
  <tabstop>versionTextBox</tabstop>
  <tabstop>btnChangeVersion</tabstop>
  <tabstop>modpackBox</tabstop>
  <tabstop>modpackEdit</tabstop>
  <tabstop>modpackBtn</tabstop>
  <tabstop>iconButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewInstanceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewInstanceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modpackBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>modpackEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modpackBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>modpackBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>versionBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>versionTextBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>versionBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnChangeVersion</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>importProviderBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>importCandidateBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
